packages/itwinui-css/src/breadcrumbs/mixins.scss

```diff
@@ -33,75 +33,39 @@
   }
 }
 
-@mixin iui-breadcrumbs-item-overrides {
-  > * {
-    padding: 0 var(--iui-size-xs);
-    overflow: hidden;
-    white-space: nowrap;
-    text-overflow: ellipsis;
-
-    &:not(.iui-button) {
-      color: var(--iui-color-text);
-    }
-  }
-
-  > :any-link {
-    @include anchor.iui-anchor;
-    box-sizing: border-box;
-    line-height: var(--iui-component-height);
-
-    &:focus {
-      outline-offset: -1px;
-    }
-  }
-
-  .iui-button {
-    &.iui-button {
-      @include borderless.iui-button-borderless;
-      border: none;
-    }
-
-    > span {
-      overflow: hidden;
-      white-space: nowrap;
-      text-overflow: ellipsis;
-    }
-
-    &:not([aria-current]) {
-      &,
-      &:hover,
-      &:active {
-        --_iui-button-text-color: var(--iui-color-text-accent);
-      }
-    }
-  }
+@mixin iui-breadcrumbs-text {
+  padding: 0 var(--iui-size-xs);
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  line-height: var(--iui-component-height);
 }
 
-@mixin iui-breadcrumbs-text {
+@mixin iui-breadcrumbs-action {
   padding: 0 var(--iui-size-xs);
```

Comment:
Since these attributes shared between both `a` and `button` should be used on any element regardless of type, should we use `iui-breadcrumbs-action` for the `<span>` examples and just get rid of `iui-breadcrumbs-text` entirely? The only issue I see with this is that the name `iui-breadcrumbs-action` would be misleading if the class was applied to a `span`.

Reply:
we could change the class name, so that's not an issue. But i feel like `<span>` has different styling - it is not blue. So we shouldn't use same class. But we can reuse styles like this:

```scss
.iui-breacrumbs-action, .iui-breadcrumbs-text {
  padding: 0 var(--iui-size-xs);
```

Reply:
This has been updated. I took a slightly different approach. I used `@include iui-breadcrumbs-text` inside of the `iui-breadcrumbs-action` mixin

Reply:
wait, so if `iui-breadcrumbs-text` is entirely duplicated inside, then we should be able to combine the two classes, no? It will greatly simplify both the css and the react code.

Try this:

```scss
.iui-breadcrumbs-item-content {
  padding: 0 var(--iui-size-xs);
  overflow: hidden;
  white-space: nowrap;
  text-overflow: ellipsis;
  line-height: var(--iui-component-height);

  &:where(a) {
    // ...
  }

  &:where(button) {
    // ...
  }
}
```

```jsx
const BreadcrumbsItem = React.forwardRef((props, forwardedRef) => {
  const { children: childrenProp, className, ...rest } = props;

  const defaultAs = !!props.href ? 'a' : !!props.onClick ? 'button' : 'span';
  const children =
    defaultAs === 'button' ? <span>{childrenProp}</span> : childrenProp;

  return (
    <Box
      as={defaultAs as 'a'}
      className={cx('iui-breadcrumbs-item-content', className)}
      ref={forwardedRef}
      {...rest}
    >
      {children}
    </Box>
  );
}) as PolymorphicForwardRefComponent<'a'>;
```

Reply:
This has been changed